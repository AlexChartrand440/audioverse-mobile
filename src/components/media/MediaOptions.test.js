import React from 'react'
import MediaOptions from './MediaOptions'

import renderer from 'react-test-renderer'

test('renders correctly', () => {
  const tree = renderer.create(<MediaOptions rate={1} onSetRate={jest.fn()} />).toJSON()
  expect(tree).toMatchSnapshot()
})
