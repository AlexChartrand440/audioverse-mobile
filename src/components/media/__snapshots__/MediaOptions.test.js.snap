// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "justifyContent": "space-around",
      "marginBottom": 20,
    }
  }
>
  <Text
    allowFontScaling={false}
    onPress={[Function]}
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 12,
        },
        Object {
          "color": "#FFFFFF",
          "fontSize": 24,
          "paddingHorizontal": 15,
        },
        Object {
          "fontFamily": "Feather",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
      ]
    }
  >
    
  </Text>
  <Text
    allowFontScaling={false}
    onPress={[Function]}
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 12,
        },
        Object {
          "color": "#FFFFFF",
          "fontSize": 24,
          "paddingHorizontal": 15,
        },
        Object {
          "fontFamily": "Feather",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
      ]
    }
  >
    
  </Text>
  <Text
    allowFontScaling={false}
    onPress={[Function]}
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 12,
        },
        Object {
          "color": "#FFFFFF",
          "fontSize": 24,
          "paddingHorizontal": 15,
        },
        Object {
          "fontFamily": "Feather",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
      ]
    }
  >
    
  </Text>
  <Text
    onPress={[MockFunction]}
    style={
      Array [
        Object {
          "color": "#FFFFFF",
          "fontSize": 24,
          "paddingHorizontal": 15,
        },
        Object {
          "fontSize": 20,
          "textAlign": "center",
          "width": 84,
        },
      ]
    }
  >
    1X
  </Text>
  <Text
    allowFontScaling={false}
    onPress={[Function]}
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 12,
        },
        Object {
          "color": "#FFFFFF",
          "fontSize": 24,
          "paddingHorizontal": 15,
        },
        Object {
          "fontFamily": "Feather",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
      ]
    }
  >
    
  </Text>
  <Text
    allowFontScaling={false}
    onPress={[Function]}
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 12,
        },
        Object {
          "color": "#FFFFFF",
          "fontSize": 24,
          "paddingHorizontal": 15,
        },
        Object {
          "fontFamily": "Feather",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
      ]
    }
  >
    
  </Text>
  <Text
    allowFontScaling={false}
    onPress={[Function]}
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 12,
        },
        Object {
          "color": "#FFFFFF",
          "fontSize": 24,
          "paddingHorizontal": 15,
        },
        Object {
          "fontFamily": "Feather",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
      ]
    }
  >
    
  </Text>
</View>
`;
